src/main/java/urlshortener/domain/safebrowsing/SBClient.java

```diff
@@ -3,49 +3,18 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.AllArgsConstructor;
+import lombok.Data;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonPropertyOrder({"clientId", "clientVersion"})
+@Data
+@AllArgsConstructor
```

Comment:
Is this creating all the getters and setters?

Reply:
Yes, see https://projectlombok.org/features/all for more information.